alphabase/statistics/regression.py

```diff
@@ -0,0 +1,235 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: ../../nbdev_nbs/statistics/regression.ipynb.
+
+# %% auto 0
+__all__ = ['EPSILON', 'LOESSRegression']
+
+# %% ../../nbdev_nbs/statistics/regression.ipynb 2
+import numpy as np
+
+from sklearn.preprocessing import PolynomialFeatures
+from sklearn.base import BaseEstimator, RegressorMixin
+
+from sklearn.utils.estimator_checks import check_estimator
+import matplotlib.pyplot as plt
```

Comment:
matplotlib is not in requirements of alphabase, as it is only for testing, put it into a hide cell `#| hide ...`